### PR TITLE
Added validation against fee

### PR DESCRIPTION
**Pull request by vanruch**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] index.js exports updated
- [x] Automatic tests added/updated
- [x] Automatic tests pass
- [x] Sanity check passed
- [x] Self-review passed
- [ ] Version updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "ambrosus-node-contracts",
   "author": "Ambrosus",
   "description": "Smart contracts used in AMB-NET",
-  "version": "0.0.26",
+  "version": "0.0.27",
```
Comment (spherefoundry) [inline package.json]: Please update this one more time when you rebase on master (0.0.27 is already taken)
